### ISSUE TITLE
can't install

### ISSUE THREAD

**Issue by JadeSAL** [wong2/chat-gpt-google-extension#15]
未能成功加载扩展程序
清单文件缺失或不可读取
无法加载清单

**Comment by wong2** [comment]
Are you installing from the build.zip file in Github Releases?

**Comment by JadeSAL** [comment]
> 
yep and I download it 3 times and tried zip and unzip files, none of it worked.

**Comment by JadeSAL** [comment]
我在chrome拉进去安装的，之前离线安装的十多个插件都成功也没有这个问题，更新了chrome也一样还是装不了

**Comment by wong2** [comment]
That's weird, can you attach the zip file here in this comment?

**Comment by southbun** [comment]
I cannot install too, edge for mac.

**Comment by ysj151215** [comment]
You should (at least) list the following to let maintainer reproduce/debug:
- Chrome version
- Extension version
- Operating system's name and its version
- Screenshot or screen recording is welcomed!

**Comment by zimengxueying** [comment]
I have met the same problem ! 
My edge vestion is 107.0.1418.62 (正式版本) (64 位)
OS version: win10
Extension version: v1.1.0
And I have solved the problem by trying 3 or more times about "Drag the unzipped folder anywhere on the page to import it "the last time I drag the file to the space named "来自其他源". Do not know it can whether solve your problem....

**Comment by Zwe1** [comment]
I've met the same problem too!

**Comment by wong2** [comment]
@Zwe1 Can you post the zip file here?

**Comment by Heinodaily** [comment]
[build (1).zip](https://github.com/wong2/chat-gpt-google-extension/files/10150994/build.1.zip)

**Comment by lazyq666** [comment]
I had the same problem，this is my file 
[chat-gpt-google-extension-main.zip](https://github.com/wong2/chat-gpt-google-extension/files/10151024/chat-gpt-google-extension-main.zip)

**Comment by wong2** [comment]
@lazyq666 Your file is wrong, please download the `build.zip` file from https://github.com/wong2/chat-gpt-google-extension/releases

**Comment by wong2** [comment]
Oh, I didn't realize that some decompress program will generate nested folders

**Comment by lazyq666** [comment]
> @lazyq666 Your file is wrong, please download the `build.zip` file from https://github.com/wong2/chat-gpt-google-extension/releases

thanks~

**Comment by JadeSAL** [comment]
> @lazyq666 Your file is wrong, please download the `build.zip` file from https://github.com/wong2/chat-gpt-google-extension/releases

ok now, thanks!
